### DISCUSSION TITLE
log

### DISCUSSION THREAD

**Discussion by MehanikTMYT** [lllyasviel/Fooocus#3092] [Q&A] [answered]
how to disable this? 

Metadata is embedded if enabled in the config or developer debug mode. You can find the information for each image in line Metadata Scheme.
And how to save history of logs (after reboot for example)?

**Comment by mashb1t** [comment, marked as answer]
metadata is disabled unless enabled. Use `--disable-metadata` to force disable the activation in the UI.
history log is enabled unless disabled. Use `--disable-image-log` to disable.
